### PR TITLE
docs: Fix grammatical errors in README

### PR DESCRIPTION
**Pull request by chaman-1**
# Fixes # 

### Checklist
- [ X] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [X ] I have read the [Contribution Guide] and my PR follows them.
- [X ] I updated my branch with the master branch.
- [X ] I have added tests that prove my fix is effective or that my feature works
- [ X] I have added necessary documentation about the functionality in the appropriate .md file
- [X ] I have added inline documentation to the code I modified

### Short description of what this PR does:
- Fixed two gramatical errors in the README.md file
- 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ This library is broken up into several packages as a monorepo so that you only n
 
 All updates to this library are documented in our [CHANGELOG](https://github.com/sendgrid/sendgrid-nodejs/blob/master/CHANGELOG.md) and [releases](https://github.com/sendgrid/sendgrid-nodejs/releases). You may also subscribe to email [release notifications](https://dx.sendgrid.com/newsletter/nodejs) for releases and breaking changes.
 
-Join an experienced and passionate team that focuses on making an impact. [Opportunities abound](https://sendgrid.com/careers) to grow the product - and grow your career! 
+Join the experienced and passionate team that focuses on making an impact. [Opportunities abound](https://sendgrid.com/careers) to grow the product - and grow your career!
```
Comment (thinkingserious) [inline README.md]: In this case, I think "an" reads better. Please revert this change. Thanks!

**Comment by thinkingserious** [comment]
Hello @chaman-k,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @chaman-k,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 1 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

File: README.md (hunk 1 above)
Comment (chaman-k) [reply README.md]: 'the' is used before definite articles, since the team mentioned above is specific(Sendgrid's team), I believe 'the' is appropriate.

**Comment by thinkingserious** [comment]
Hello @chaman-1,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX
